docs/app/pages/components/components-sections/drag-and-drop/drag-and-drop-cards/snippets/app.css

```diff
@@ -38,16 +40,20 @@
     outline-offset: 0;
 }
 
-.dnd-cards-container > .card.gu-mirror {
+.card.cdk-drag-preview {
```

Comment:
I'm not totally sure what the best answer on this is because obviously the old styles wont work because dragula has been removed so if I don't change the styles then they wont appear properly or mirror what was there before.

Reply:
Just to recap what was said in standup, Ashley suggested that `gu-mirror` could be added to `previewClass`.